utils/mock-data-loader/user-helper.js

```diff
@@ -24,43 +26,57 @@ const mockDataLoaderTFMUser = {
 };
 
 const createAndLogInAsInitialUser = async () => {
-  console.info('Portal login as user %s', mockDataLoaderUser.username);
+  logger.info('logging in as initial portal user');
   try {
     return await api.loginViaPortal(mockDataLoaderUser);
-  } catch {
-    console.info('Creating portal user %s', mockDataLoaderUser.username);
+  } catch (error) {
+    logger.info(
+      `login failed for initial portal user:\
+    \n    ${error.message}\nrecreating initial portal user`,
+      { colour: LOGGER_COLOURS.yellow, depth: 1 },
+    );
     await api.createInitialUser(mockDataLoaderUser);
     return api.loginViaPortal(mockDataLoaderUser);
   }
 };
 
 const deleteInitialUser = async (token) => {
+  logger.info('deleting initial Portal user');
   const allUsers = await api.listUsers(token);
   const userToDelete = allUsers.find((user) => user.username === mockDataLoaderUser.username);
-  console.info(`deleting user ${userToDelete.username}`);
   await api.deleteUser(userToDelete, token);
 };
 
 const createAndLogInAsInitialTfmUser = async () => {
-  console.info('TFM login as user %s', mockDataLoaderTFMUser.username);
-  let token = await api.loginTfmUser(mockDataLoaderTFMUser);
+  try {
+    logger.info('logging in as initial TFM user');
+    const token = await api.loginTfmUser(mockDataLoaderTFMUser);
 
-  if (!token) {
-    console.info('Creating TFM user %s', mockDataLoaderTFMUser.username);
+    if (!token) {
+      throw new FailedToCreateLoggedInUserSessionError({ username: mockDataLoaderTFMUser.username, cause: 'No token was present on response' });
+    }
 
+    return token;
+  } catch (error) {
+    logger.info(
+      `login failed for initial TFM user:\
+        \n    ${error.message}\nrecreating initial TFM user`,
+      {
+        depth: 1,
+        colour: LOGGER_COLOURS.yellow,
```

Comment:
I'm wondering if, instead of referring to an actual colour we have something like `CATCH` - that refers to X colour.

Otherwise seems it could be easy for someone to _not_ have yellow in a catch handler and then would be misaligned, potentially confusing when debugging.

What do you think?